dlt/load/utils.py

```diff
@@ -158,12 +162,17 @@ def _init_dataset_and_update_schema(
     )
     job_client.initialize_storage()
     if drop_tables:
-        drop_table_names = [table["name"] for table in drop_tables]
         if hasattr(job_client, "drop_tables"):
             logger.info(
-                f"Client for {job_client.config.destination_type} will drop tables {staging_text}"
+                f"Client for {job_client.config.destination_type} will drop tables"
+                f" {drop_tables} {staging_text}"
+            )
+            job_client.drop_tables(*drop_tables, delete_schema=True)
+        else:
+            logger.warning(
```

Comment:
which client does not support this? I thought this was a universal interface, it's implemented for filesystem too.

Reply:
ah ok, the custom destination..

Reply:
vector databases as well